string RA
string DEC
string Sidereal
string azimuth
string altitud
string slew_rate
string msg
